polygeist/tools/cgeist/Lib/clang-mlir.cc

```diff
@@ -3266,6 +3276,7 @@ void MLIRASTConsumer::createMLIRParameterDescriptors(
     const clang::FunctionDecl &FD,
     SmallVectorImpl<CodeGenUtils::ParmDesc> &parmDescriptors) {
   assert(parmDescriptors.empty() && "Expecting 'parmDescriptors' to be empty");
+  llvm::dbgs() << "\n-- Entering createMLIRParameterDescriptors --\n";
```

Comment:
This is displaying a message every time this function is called. Please, wrap around `LLVM_DEBUG`.

Reply:
Done in https://github.com/intel/llvm/pull/7149.